src/components/Game/Game.js

```diff
@@ -29,9 +31,11 @@ export default class Game extends Component {
 
   componentDidMount() {
     const screenWidth = window.innerWidth - 1; // this is important for mobile to fit on screen
+    const vimMode = JSON.parse(localStorage.getItem(vimModeKey)) || false;
```

Comment:
this might need a try catch just to be safe -- otherwise a bad value in localStorage could brick the site

Reply:
good point! added the try/catch